src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/ResourceDeserializer.java

```diff
@@ -15,28 +15,17 @@
 import com.fasterxml.jackson.databind.deser.std.StdNodeBasedDeserializer;
 import com.fasterxml.jackson.databind.type.TypeFactory;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Iterator;
 import java.util.List;
 
-public class ResourceDeserializer extends StdNodeBasedDeserializer<List<Resource>> {
+public class ResourceDeserializer extends StdNodeBasedDeserializer<Resource> {
     public ResourceDeserializer() {
         super(TypeFactory.defaultInstance().constructCollectionType(List.class, Resource.class));
```

Comment:
I don't think this is required anymore as this deserializer does not return list anymore